src/frontend/komponenter/Behandling/Vurdering/VurderingUtil.ts

```diff
@@ -36,11 +36,7 @@ export const harBesvartPåAlleDelvilkår = (delvilkårsvurderinger: IDelvilkår[
 export const skalViseLagreKnapp = (vurdering: IVurdering, config: IVilkårConfig): boolean => {
     const { begrunnelse, delvilkårsvurderinger } = vurdering;
 
-    const begrunnlsePåkrevd =
-        config.begrunnelsePåkrevdHvisOppfylt === undefined
-            ? true
-            : config.begrunnelsePåkrevdHvisOppfylt;
-    if (begrunnlsePåkrevd && manglerBegrunnelse(begrunnelse)) {
+    if (config.begrunnelsePåkrevdHvisOppfylt && manglerBegrunnelse(begrunnelse)) {
```

Comment:
Her gjør du en endring i logikken - er det meningen? Den kan kanskje bare være true/false nå? Isåfall - bra! 👌

Reply:
Ja, det var meningen. Bare true/false nå 👌